launcher/launcher/main.go

```diff
@@ -35,13 +35,22 @@ var launchSpec spec.LaunchSpec
 
 func main() {
 	var exitCode int
-	defer func() {
-		os.Exit(exitCode)
-	}()
 
 	logger = log.Default()
+	// default logger output to stderr
```

Comment:
stdout or stderr?

Reply:
log.Default() will print to stderr, I want it to output to stdout. I will make the comment more clear.

Reply:
any reason we want stdout instead

Reply:
no particular, most of the log we output info, so stdout seems like a better fit.